### DISCUSSION TITLE
questions in the first createArrowFiles

### DISCUSSION THREAD

**Discussion by 1221li** [GreenleafLab/ArchR#1151] [Questions / Documentation]
In my fragment file, there are more than 300,000 barcodes after removing the duplicates. However, I imported the fragment file into archR and created the arrow file. Without adding any filtering conditions, there are only more than two thousand cells. I do not understand this, and I hope you can provide me with any suggestions about how do I get all barcodes in the Arrow file

ArrowFiles <- createArrowFiles(inputFiles = inputFiles,sampleNames = names(inputFiles),filterTSS = NULL,filterFrags = NULL,addTileMat = TRUE,addGeneScoreMat = TRUE,geneAnnotation = geneAnnotation,genomeAnnotation = genomeAnnotation)

**Comment by rcorces** [comment]
`filterFrags` and `filterTSS` are deprecated. You need to use `minFrags` and `minTSS`. This is somewhat reflected in the function documentation as `filterFrags` and `filterTSS` are no longer defined.
https://www.archrproject.com/reference/createArrowFiles.html

https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/CreateArrow.R#L107

**Reply by 1221li** [reply]
Thank you for your reply. I have another question about "Pairwise Testing Between Groups".
markerTest <- getMarkerFeatures(
  ArchRProj = projHeme5, 
  useMatrix = "PeakMatrix",
  groupBy = "Clusters2",
  testMethod = "wilcoxon",
  bias = c("TSSEnrichment", "log10(nFrags)"),
  useGroups = "Erythroid",
  bgdGroups = "Progenitor"
)
I can get the genelist in useGroup with this code, but I don't know how to get the gebelist in bgdGroup 。What should I do if I want to get gene list from bgdgroup。
I look forward to your reply！

**Reply by rcorces** [reply]
Its beneficial to not ask two questions in the same thread because this makes it harder for others to find previous questions.

To answer your question, I think there is some confusion on useGroups and bgdGroups. These are groups of cells, not groups of genes.
